x-pack/plugins/osquery/public/common/hooks/use_error_toast.tsx

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ErrorToastOptions, IToasts, Toast } from 'kibana/public';
+import { useState } from 'react';
+
+export const useErrorToast = (toasts: IToasts) => {
```

Comment:
can we instead just use `useKibana()` inside `useErrorToast` to avoid passing `toasts` param every time?